reproman/support/jobs/tests/test_orchestrators.py

```diff
@@ -433,18 +433,65 @@ def test_orc_datalad_run_container(tmpdir, dataset,
 
 
 @pytest.mark.integration
-def test_orc_datalad_pair(job_spec, dataset, shell):
+@pytest.mark.parametrize("orc_class",
+                         [orcs.DataladPairOrchestrator,
+                          orcs.DataladNoRemoteOrchestrator],
+                         ids=["orc:pair", "orc:no-remote"])
+def test_orc_datalad_nonrun(job_spec, dataset, shell, orc_class):
     with chpwd(dataset.path):
-        orc = orcs.DataladPairOrchestrator(
-            shell, submission_type="local", job_spec=job_spec)
+        orc = orc_class(shell, submission_type="local", job_spec=job_spec)
         orc.prepare_remote()
         orc.submit()
         orc.follow()
 
         orc.fetch()
-        # The local fetch variant doesn't currently get the content, so just
-        # check that the file is under annex.
         assert dataset.repo.is_under_annex("out")
+        assert (dataset.pathobj / "out").exists()
+
+
+@mark.skipif_no_datalad
+@pytest.mark.integration
+@pytest.mark.parametrize("should_pass", [True, False], ids=["success", "failure"])
+def test_orc_datalad_no_remote_get(tmpdir, shell, should_pass):
+    import datalad.api as dl
+
+    topdir = str(tmpdir)
+    ds_a = dl.create(op.join(topdir, "a"))
+    if should_pass:
+        (ds_a.pathobj / "foo").write_text("data")
+        ds_a.save()
+
+    ds_b = dl.clone(ds_a.path, op.join(topdir, "b"))
+    assert not ds_b.repo.file_has_content("foo")
+    with chpwd(ds_b.path):
+        orc = orcs.DataladNoRemoteOrchestrator(
+            shell, submission_type="local",
+            job_spec={"root_directory": op.join(topdir, "nm-run"),
```

Comment:
What is this nm-run subdirectory? Or nm is for "never mind" , is could be anything?

Reply:
It's a poorly chosen name copied over from the `job_spec` fixture, where "nm" was for "niceman".  Renamed in c67e828f9.